datacommons/node.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" API to request node information.
+"""
+
+from typing import Dict, List
+
+from datacommons.requests import _post
+from datacommons.utils import _get_direction
+
+
+def properties(nodes: List[str], out: bool = True) -> Dict[str, List[str]]:
+    """Retrieves all the properties for a list of nodes.
```

Comment:
I'm not as familiar with pypi - if the docstring gets converted into documentation, do we need to put a Returns here so users know what to expect?

Reply:
I think these docs are only meant for code, not for external documentations. I will put a short blurb of the return values